### PR TITLE
[PATCH v4] changelog: updates for odp v1.20.0.0

### PR DESCRIPTION
**Pull request by Bill-Fischofer-Linaro**
Add updates for ODP v1.20.0.0

Signed-off-by: Bill Fischofer <bill.fischofer@linaro.org>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,312 @@
...
+The new scheduler APIs `odp_schedule_multi_wait()` and
+`odp_schedule_multi_no_wait()` are added to provide more efficiently
+implementable versions of these functions. The existing scheduler APIs remain
+unchanged. These new APIs can simply provide a fastpath for some
+applications/implementations as an alternative to specifying a parameter on
+`odp_schedule_multi()`.
+
+===== Move Queue Scheduler Capabilities to Scheduler
+As part of the restructure of the scheduler APIs, queue-related scheduling
+capabilities are moved from the `odp_queue_capability_t` struct returned by
```
Comment (matiaselo) [inline CHANGELOG]: `odp_schedule_capability()` is not part of this release.

**Comment by matiaselo** [comment]
Should the helper changes be mentioned?

E.g.
- Removed odph_odpthread_linuxtype_t and replaced it with the newly added odp_mem_model_t
- Added odph_options() for reading parsed linux helper options

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v2:
* Rebased on merged master
* Incorporate review changes suggested by @matiaselo

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,311 @@
...
+when this field is needed and how it should be used.
+
+===== Classifier Type Rename
+The `odp_pmr_t` type name for an invalid value is renamed from `ODP_PMR_INVAL`
+to `ODP_PMR_INVALID` for consistency with the rest of ODP type names. The old
+symbol is still available when ODP is configured with 
+`--enable-deprecated`.
+
+===== New API for Packet Event Subtypes
+The `odp_event_subtype()` API is added that returns the subtype of a packet
```
Comment (lumag) [inline CHANGELOG]: `odp_packet_subtype()`

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,311 @@
...
+==== Native Builds
+When running in non-ABI compatibility mode, `odp-linux` now enables
+native machine-specific optimizations for the CPU architecture of the
+local machine.
+
+=== Validation Test Improvements
+==== SCTP Test Packets
+SCTP test packets are now used in parser testing. SCTP headers are added to
+ODP and ODP helpers and SCTP checksums are now inserted and verified as part
+of validation testing.
```
Comment (lumag) [inline CHANGELOG]: Addition of `odph_sctp_chksum_set ( odp_packet_t pkt )` and `odph_sctp_chksum_verify ( odp_packet_t pkt )` should be mentioned somewhere.

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v3:
* Rebased
* Incorporate @lumag review comments and suggestions.

File: CHANGELOG (hunk 3 above)
Comment (Bill-Fischofer-Linaro) [reply CHANGELOG]: OK, v3 adds a Helper section to cover these. Thanks.

File: CHANGELOG (hunk 2 above)
Comment (Bill-Fischofer-Linaro) [reply CHANGELOG]: Fixed in v3.

**Comment by lumag** [comment]
Reviewed-by: Dmitry Eremin-Solenikov

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,315 @@
...
+* `ODP_AUTH_ALG_SNOW3G_UIA2`
+* `ODP_AUTH_ALG_ZUC_EIA3`
+
+These values are returned as ODP capabilities as well as being accepted in
+crypto session creation for implementations that indicate support for them.
+
+===== Crypto Capability for Bitwise Operation
+The new `bit_mode` capability Boolean is added to the
+`odp_crypto_cipher_capability_t` struct to indicate that an implementation
+supports operating supporting algorithms in bit mode. When operating in bit
```
Comment (muvarov) [inline CHANGELOG]: "an implementation ￼ supports bit mode  algorithms."  is more simple.

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,315 @@
...
+mode, field offsets and lengths are expressed in terms of bits rather than
+bytes. However, such lengths must always be specified in multiples of 8.
+
+===== Improved Crypto Spec Documentation
+The ODP crypto API specification is tightened to specify default values for
+cipher and authentication algorithms. Also documented wen key and IV
```
Comment (muvarov) [inline CHANGELOG]: wen -> when

**Comment by Bill-Fischofer-Linaro** [comment]
Changes for v4
* Rebased
* Fix typos as suggested by @muvarov

**Comment by muvarov** [comment]
Merged.

**Pull request closed without merging**